Opserver/Content/js/Scripts.js

```diff
@@ -800,7 +800,7 @@ Status.SQL = (function () {
                 Status.popup('sql/active/filters' + window.location.search, null, filterOptions);
             },
             '#/db/': function (val, firstLoad, prev) {
-                var obj = val.indexOf('tables/') > 0 || val.indexOf('views/') || val.indexOf('storedprocedures/') || val.indexOf('unusedindexes/') > 0 || val.indexOf('querystore/') > 0
+                var obj = val.indexOf('tables/') > 0 || val.indexOf('views/') || val.indexOf('storedprocedures/') || val.indexOf('unusedindexes/') > 0 || val.indexOf('querystore/') > 0 || val.indexOf('blitzcache/') > 0
```

Comment:
not your code, but wow that's hacky :)

Reply:
theres a comment somewhere around there about making it better...